app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -235,7 +235,7 @@
     public static final String IS_MUTED = "is_muted";
 
     /*//////////////////////////////////////////////////////////////////////////
-    // Time constants
+    Time constants
```

Comment:
With this change the comment is the odd one out if you compare it to the other ones

Reply:
Ok, sorry for this mistake, I rollbacked this change to keep the form of commentaire same